src/commands/play.rs

```diff
@@ -365,10 +365,10 @@ async fn insert_track(
     let queue_size = handler.queue().len();
     drop(handler);
 
-    verify(
-        idx != 0 && idx < queue_size,
-        ParrotError::NotInRange("index", idx as isize, 1, queue_size as isize),
-    )?;
```

Comment:
Yeah but insert 5 on a queue of length 3 now is not checked right?

Reply:
Correct, but looking at these function calls I can't really see how that scenario is possible. Am I overlooking something?

Reply:
If insert is called with our index command?

Reply:
You mean a command for inserting a track in a specific position in the queue? We don't have that.

Reply:
Added the assertion back. Idx is always >=1, so having the assert before the new code would fail (1 <= 0). So I added it as a safeguard for the already existing one.